proof-systems/src/darlin/tests/mod.rs

```diff
@@ -466,6 +543,7 @@ mod test {
             vks.as_slice(),
             &verifier_key_g1,
             &verifier_key_g2,
+            None,
```

Comment:
Doing the same as previous tests also here won't harm

Reply:
Done